### ISSUE TITLE
Framework core - single column and single width submit / show feedback button layout

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#2878]
### Subject of the enhancement
The question submit (show model answer) and show feedback buttons have an inefficient layout for longer text (such as translations) in the single column (mobile view) and single width (left / right in desktop view).

I propose a change to both the single column and single width view that would set the submit and show feedbacks buttons to fill the width and introduce a gap in between for the marking to appear. I welcome feedback and thoughts on this idea.

Screenshots below.

### Your environment
* Latest FW

### Single column (mobile)

Current view EN:
![button-03](https://user-images.githubusercontent.com/11569678/90029643-033ff000-dcb3-11ea-99b7-59fe3312d05a.png)

Proposed view EN:
![button-06](https://user-images.githubusercontent.com/11569678/90029698-0fc44880-dcb3-11ea-816a-c59f46f7cfdf.png)

### Single width (desktop)

Current view EN:
![button-01](https://user-images.githubusercontent.com/11569678/90029438-caa01680-dcb2-11ea-9f6c-54b9485785df.png)

Current view FR:
![button-01a](https://user-images.githubusercontent.com/11569678/90029479-d4297e80-dcb2-11ea-9e26-cd93dd8786c4.png)

Proposed view EN:
![button-04](https://user-images.githubusercontent.com/11569678/90029527-df7caa00-dcb2-11ea-8470-fa6120f7d4d1.png)

Proposed view FR:
![button-04a](https://user-images.githubusercontent.com/11569678/90029518-dd1a5000-dcb2-11ea-9eb4-70aade47ad64.png)

**Comment by moloko** [comment]
could the tick/cross icon between the submit and f/b btns be smaller, to allow the gap between the two buttons to be closer to the gap between the last answer option and the submit button? perhaps make that gap slightly larger also?

Equally would showing it as an icon in the submit button itself be an option?

**Comment by moloko** [comment]
I agree wholeheartedly with the idea of changing the layout to allow for more room for localisation though

**Comment by moloko** [comment]
Looking at this more closely has made me realise that we should probably re-label that button to 'Show correct answer' or 'View correct answer' to make it crystal clear that an action will take place when it is clicked...

**Comment by oliverfoster** [comment]
@guywillis any further thoughts on this one?
